tests/yast2_cmd/yast_rdp.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: this test checks that YaST2's RDP module is behaving
+#          correctly in cmdline mode  and verifying that RDP service
+#          has been successfully set.
+# Maintainer: Jun Wang <jgwang@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils 'is_sle';
+
+sub run {
+    select_console 'root-console';
+
+    # yast2-rdp was added after SLE12SP2.
+    if (is_sle('15+')) {
```

Comment:
If this feature is added after SLE12SP2, than SLE12SP3 if is supported? If yes, this condition might need a branch for SLE12 product line.

Reply:
I have corrected it. Thank you.

Reply:
I have corrected it. Thank you.